src/Uno.UI/UI/Xaml/Shapes/ArbitraryShapeBase.Android.cs

```diff
@@ -116,8 +116,10 @@ private IDisposable BuildDrawableLayer()
             var imageBrushFill = Fill as ImageBrush;
             if (imageBrushFill != null)
             {
-					var bitmapDrawable = new BitmapDrawable(imageBrushFill.TryGetBitmap(drawArea, () => RefreshShape(forceRefresh: true), path));
-					drawables.Add(bitmapDrawable);
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
It's fine to hide warnings in legacy code, but we shouldn't be doing it for current code... there's no way to address the warning?